dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/internal/FeatureResolverImpl.java

```diff
@@ -831,7 +842,9 @@ void primeSelected(Collection<String> features) {
             }
             // remove any conflicts recorded above
             for (Map.Entry<String, String> conflict : conflicts.entrySet()) {
-                features.remove(conflict.getKey());
+                if (!allowMultipleVersions(conflict.getValue())) {
+                    features.remove(conflict.getKey());
+                }
                 _current._selected.remove(conflict.getValue());
```

Comment:
I'm not sure why we remove the base name as selected when an allowed conflict is found.

Reply:
This is primeSelected.  Since there is an allowedConflict, we cannot select just one of them which is hy it is removed until it can be selected with the rest of resolution logic.  Without this logic microprofile 4 packaging wouldn't work since the only features there are microProfile-4.0 and microProfile-4.1.

Reply:
Updated to not have this logic any longer and changed to use continue.